src/App.js

```diff
@@ -51,30 +51,103 @@ function App() {
   }, [todoList, isLoading]);
 
 
-  const addTodo = newTodo => {
+  const addTodo = async(newTodo) => {
     // setTodoList ([...todoList, newTodo]) USE PREVTODO!!!
-    setTodoList(prevTodoList => ([ ...prevTodoList, newTodo ]))
+   // setTodoList(prevTodoList => ([ ...prevTodoList, newTodo ]))
+   await postTodo(newTodo);
+  
   }  
+  const postTodo = async (newTodo) => {
+    try {
+      const airtableData = {
+        fields: {
+          title: newTodo.title
+        },
+      };
+  
+      const response = await fetch(
+        `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}/`,
```

Comment:
Nitpick:
I'd recommend moving this URL into a constant below the `import` statements. This way, you don't have to re-write it multiple times:
```js
const AIRTABLE_API_URL = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}/`

...etc

fetch(AIRTABLE_API_URL, { /* fetch options */ });
```